tests/tensorflow/test_tensorflow_model_export.py

```diff
@@ -236,32 +236,29 @@ def test_save_model_with_invalid_path_signature_def_or_metagraph_tags_throws_exc
         tmpdir, saved_tf_iris_model):
     model_path = os.path.join(str(tmpdir), "model")
 
-    with pytest.raises(IOError) as e:
+    with pytest.raises(IOError):
         mlflow.tensorflow.save_model(tf_saved_model_dir="not_a_valid_tf_model_dir",
                                      tf_meta_graph_tags=saved_tf_iris_model.meta_graph_tags,
                                      tf_signature_def_key=saved_tf_iris_model.signature_def_key,
                                      path=model_path)
 
-    with pytest.raises(RuntimeError) as e:
+    with pytest.raises(RuntimeError):
         mlflow.tensorflow.save_model(tf_saved_model_dir=saved_tf_iris_model.path,
                                      tf_meta_graph_tags=["bad tags"],
                                      tf_signature_def_key=saved_tf_iris_model.signature_def_key,
                                      path=model_path)
-        assert e.error_code == INVALID_PARAMETER_VALUE
```

Comment:
Removed this since RuntimeError (and other built-in Python exceptions) don't have an `error_code` attribute. These tests passed before since Pytest never actually reaches this line (one the RuntimeError is thrown, we exit out of the `with` block). I confirmed this by adding an `assert False` after this line while still within the `with` block - the test still passes

Reply:
Good catch! Thanks!